operator/src/main/resources/scripts/startNodeManager.sh

```diff
@@ -246,16 +246,13 @@ EOF
   [ ! $? -eq 0 ] && trace "Failed to create '${wl_props_file}'." && exit 1
 
   if [ ! "${ADMIN_NAME}" = "${SERVER_NAME}" ]; then
-    admin_protocol="http"
-    if [ "${ADMIN_PORT_SECURE}" = "true" ]; then
-      admin_protocol="https"
-    fi  
-    echo "AdminURL=$admin_protocol\\://${AS_SERVICE_NAME}\\:${ADMIN_PORT}" >> ${wl_props_file}
+    ADMIN_URL=$(getAdminUrl)
+    export ADMIN_URL
+    trace "ADMIN_URL"=$ADMIN_URL >> ${wl_props_file}
```

Comment:
Is any other script using the ADMIN_URL env var?  If not, it shouldn't be exported.

The props file entry needs to be in a special format that's acceptable to the NM, so line 251 should be:
  echo "AdminURL=$ADMIN_URL" >> ${wl_props_file}
or even
  echo "AdminURL=$(getAdminUrl)" >>  ${wl_props_file}
not
  trace "ADMIN_URL"=$ADMIN_URL >> ${wl_props_file}